docs/script.js

```diff
@@ -1,5 +1,5 @@
-const navMenuButton = document.getElementById("coffee-menu");
-const menuButton = document.getElementById("button-icon-burger");
+// const navMenuButton = document.getElementById("coffee-menu");
+// const menuButton = document.getElementById("button-icon-burger");
 const navModal = document.getElementById("navModal");
 const sliderContainer = document.getElementById("sliderContainer");
 let indexValue = 1;
```

Comment:
Is this file neccessary?

Reply:
No that's the JS file I used in the previous chapter. I'll remove them if we don't need them anymore.